src/main/java/com/academy/terai/TeraiApplication.java

```diff
@@ -1,14 +1,19 @@
 package com.academy.terai;
-
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
 
 
 @SpringBootApplication(exclude={DataSourceAutoConfiguration.class})
-
+@ComponentScan({"com.academy.terai.Controller"})
```

Comment:
ComponentScan should be needed only if your main class, this one, doesn't contain in those classes in its package or subpackages. Didn't it work without it?